src/client/input/header/gyrospace.h

```diff
@@ -0,0 +1,393 @@
+/*
+ * =======================================================================
+ *
+ * Gyro Space to Play - A plug-and-play Gyro Space Transformer code
+ * Version 0.8.5
+ *
+ * Provides functionality for transforming gyro inputs into Local Space,
+ * Player Space, and World Space, while handling sensitivity adjustments,
+ * gravity alignment and dynamic orientation.
+ *  
+ * Compatible with both C and C++ environments.
+ *
+ * Based on the work by Jibb Smart (JoyShockMapper, GamepadMotionHelpers,
+ * Fortnite v.19.30's Gyro Aim/Flick Stick implementation)
+ *
+ * =======================================================================
+ */
+
+ #ifndef GYROSPACE_HPP
+ #define GYROSPACE_HPP
+ 
+ #ifdef __cplusplus
```

Comment:
unnecessary, just use the normal C includes for C and C++.
the only c++-specific thin you need is (below the includes) 
```c
#ifdef __cplusplus
extern "C" {
#endif
```

and at the end of the file the corresponding
```c
#ifdef __cplusplus
}
#endif
```